api_FTL.php

```diff
@@ -438,4 +438,19 @@
       $data["dns-port"] = $return[0];
     }
 }
+
+if (isset($_GET['gateway']) && $auth) {
+    $return = callFTLAPI("gateway");
+    if (array_key_exists("FTLnotrunning", $return)) {
+      $data = array("FTLnotrunning" => true);
+    } else {
+      $gateway = array();
+      $tmp = explode(" ", $return[0]);
+      $gateway[] = array(
+          "ip" => $tmp[0],
+          "iface" => $tmp[1]
```

Comment:
Suggestion: 
there is a different way to do this, instead of using a temp variables:

```php
      $gateway = array_combine(["ip", "iface"], explode(" ", $return[0]);
```

Reply:
Nice. Thanks